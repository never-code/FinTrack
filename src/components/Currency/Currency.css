.currency-container {
  text-align: end;
}
.currency-container select {
  cursor: pointer;
  border: none;
  background-color: gainsboro;
  font-weight: 800;
  color: var(--text-color);
  font-size: 20px;
  outline: none;
}
