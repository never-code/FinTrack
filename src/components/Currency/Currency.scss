.currency-container {
  text-align: end;
  select {
    cursor: pointer;
    border: none;
    background-color: gainsboro;
    font-weight: 800;
    color: var(--text-color);
    font-size: 20px;
    outline: none;

  }
}