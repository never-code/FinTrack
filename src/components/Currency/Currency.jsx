import React, { useState } from 'react';
import './Currency.scss';

const CurrencyContainer = ({ change }) => {
  const [currency, setCurrency] = useState(localStorage.getItem('currency') || 'INR');
  const onChange = (v) => {
    localStorage.setItem('currency', v);
    setCurrency(v);
    change();
  };
  return (
    <div className="currency-container">
      {/* <span>&#129504;</span> */}
      <select name="Currency" onChange={({ target: { value } }) => onChange(value)} value={currency}>
        <option value="INR">INR</option>
        <option value="USD">USD</option>
        <option value="EUR">EUR</option>
        {/* <option value="CNY">CNY</option> */}
      </select>
    </div>
  );
};

export default CurrencyContainer;
