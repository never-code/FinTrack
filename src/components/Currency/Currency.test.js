import React from 'react';
import { render } from '@testing-library/react';
import Currency from './Currency';

describe('SliderContainer ', () => {
  it('should render successfully', () => {
    const { container } = render(<Currency />);
    expect(container).toMatchSnapshot();
  });
});
