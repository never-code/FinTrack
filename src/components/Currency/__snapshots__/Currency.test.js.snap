// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SliderContainer  should render successfully 1`] = `
<div>
  <div
    class="currency-container"
  >
    <span>
      ðŸ§ 
    </span>
    <select
      name="Currency"
    >
      <option
        value="INR"
      >
        INR
      </option>
      <option
        value="USD"
      >
        USD
      </option>
      <option
        value="EUR"
      >
        EUR
      </option>
    </select>
  </div>
</div>
`;
